packages/fadbadml/fadbadml.0.1.2/opam

```diff
@@ -9,7 +9,10 @@ homepage: "https://fadbadml-dev.github.io/FADBADml/"
 doc: "https://fadbadml-dev.github.io/FADBADml/doc/"
 bug-reports: "https://github.com/fadbadml-dev/FADBADml/issues"
 depends: [
+  "ocaml" {>= "4.08"}
```

Comment:
extra note for the author: backward compatibility could be kept if `stdlib-shims` was added as a dependency (but that would require to change the package to actually add it to the `(libraries ...)` fields in the dune files)